modules/reference/pages/properties/object-storage-properties.adoc

```diff
@@ -0,0 +1,1494 @@
+
+*Type:* string
+
+*Default:* `null`
+
+---
+
+=== cloud_storage_attempt_cluster_restore_on_bootstrap
+
+When set to `true`, Redpanda automatically retrieves cluster metadata from a specified object storage bucket at the cluster's first startup. This option is ideal for orchestrated deployments, such as Kubernetes. Ensure any previous cluster linked to the bucket is fully decommissioned to prevent conflicts between Tiered Storage subsystems.
+
+*Requires restart:* Yes
+
+*Optional:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* boolean
+
+*Default:* `false`
+
+---
+
+=== cloud_storage_azure_adls_endpoint
+
+Azure Data Lake Storage v2 endpoint override. Use when Hierarchical Namespaces are enabled on your storage account and you have set up a custom endpoint.
+
+*Requires restart:* Yes
+
+*Optional:* No
+
+*Visibility:* `user`
+
+*Type:* string
+
+*Default:* `null`
+
+---
+
+=== cloud_storage_azure_adls_port
+
+Azure Data Lake Storage v2 port override. Also see <<cloud_storage_azure_adls_endpoint>>. Use when Hierarchical Namespaces are enabled on your storage account and you have set up a custom endpoint.
+
+*Requires restart:* Yes
+
+*Optional:* No
+
+*Visibility:* `user`
+
+*Type:* integer
+
+*Accepted values:* [`0`, `65535`]
+
+*Default:* `null`
+
+---
+
+=== cloud_storage_azure_container
+
+The name of the Azure container to use with Tiered Storage. If `null`, the property is disabled.
+
+NOTE: The container must belong to <<cloud_storage_azure_storage_account,cloud_storage_azure_storage_account>>.
+
+*Type*: string
+
+*Default*: null
+
+*Restart required*: yes
+
+*Supported versions*: Redpanda v23.1 or later
+
+---
+
+=== cloud_storage_azure_managed_identity_id
+
+The managed identity ID to use for access to the Azure storage account. To use Azure managed identities, you must set <<cloud_storage_credentials_source>> to `azure_vm_instance_metadata`. See xref:manage:security/iam-roles.adoc[IAM Roles] for more information on managed identities.
+
+*Type*: string
+
+*Default*: null
+
+*Restart required*: no
+
+*Supported versions*: Redpanda v24.1 or later
+
+---
+
+=== cloud_storage_azure_shared_key
+
+The shared key to be used for Azure Shared Key authentication with the Azure storage account configured by <<cloud_storage_azure_storage_account,cloud_storage_azure_storage_account>>.  If `null`, the property is disabled.
+
+NOTE: Redpanda expects this key string to be Base64 encoded.
+
+*Type*: string
+
+*Default*: null
+
+*Restart required*: yes
+
+*Supported versions*: Redpanda v23.1 or later
+
+---
+
+=== cloud_storage_azure_storage_account
+
+The name of the Azure storage account to use with Tiered Storage. If `null`, the property is disabled.
+
+*Requires restart:* Yes
+
+*Optional:* No
+
+*Visibility:* `user`
+
+*Type:* string
+
+*Default:* `null`
+
+---
+
+=== cloud_storage_backend
+
+Optional object storage backend variant used to select API capabilities. If not supplied, will be inferred from other configuration parameters.
```

Comment:
```suggestion
Optional object storage backend variant used to select API capabilities. If not supplied, this will be inferred from other configuration properties.
```